SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealMetadata.h

```diff
@@ -76,19 +79,20 @@ struct UnrealMetadata : Component
 #if !UE_BUILD_SHIPPING
 		if (NativeClass.IsStale())
 		{
-			UE_LOG(LogSpatialClassInfoManager, Warning, TEXT("UnrealMetadata native class %s unloaded whilst entity in view."), *ClassPath);
+			UE_LOG(LogSpatialUnrealMetadata, Warning, TEXT("UnrealMetadata native class %s unloaded whilst entity in view."), *ClassPath);
 		}
 #endif
-		UClass* Class = nullptr;
+		UClass* Class = FindObject<UClass>(nullptr, *ClassPath, false);
```

Comment:
If we end up moving NeedToLoadClass somewhere we could add another function GetClassFromPath to be used here and also by NeedToLoadClass .